frontend/playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
```

Comment:
is it generated code? if not maybe we should delete this comments. Also idk if testing both edge and chrome is necessary since they are both on chromium as far as i know. Checking firefox/safari/opera seems to be a better idea

Reply:
I is generated code with a small change. I commented firefox + safari. These are more restrictive and for our app to work on those we need to get a 3rd party authenticated cert or run everything on http. I will set it to work on http in the next commit but i don't know which option is better